community_server/src/Controller/JsonRequestHandlerController.php

```diff
@@ -332,33 +332,72 @@ private function errorInTransaction($jsonData) {
       }
       return $this->returnJson(['state' => 'success']);
     }
+    
+    private function sendEMailTransactionFailed($transaction, $reason_type)
+    {
+        $disable_email = Configure::read('disableEmail', false);  
+        if($disable_email) {
+            return;
+        }
+        $transaction_body = $transaction->getTransactionBody();
+        $transaction_type_name = $transaction_body->getTransactionTypeName();
+        $senderUser = null;
+        if($transaction_type_name === 'transfer') {
+            $senderUser = $transaction_body->getSpecificTransaction()->getSenderUser();
+        } else if($transaction_type_name === 'creation') {
+            $senderUser = $transaction->getFirstSigningUser();
+        }
+      // send notification email
+        $noReplyEmail = Configure::read('noReplyEmail');
+        if($senderUser) {
+          try {
+            $email = new Email();
+            $emailViewBuilder = $email->viewBuilder();
+            $emailViewBuilder->setTemplate('notificationTransactionFailed')
+                             ->setVars(['user' => $senderUser, 'transaction' => $transaction, 'reason' => $reason_type]);
+            $receiverNames = $senderUser->getNames();
+            if($receiverNames == '' || $senderUser->email == '') {
+              $this->addError('TransactionCreation::sendNotificationEmail', 'to email is empty for user: ' . $senderUser->id);
+              return false;
+            }
+            $email->setFrom([$noReplyEmail => 'Gradido (nicht antworten)'])
+                  ->setTo([$senderUser->email => $senderUser->getNames()])
+                  ->setSubject(__('Gradido Transaktion fehlgeschlagen!'))
```

Comment:
What about localization? Can we send Emails in other languages than German?

Reply:
Was steht denn im Body? Gibt es einen Hinweis welche Transaktion fehlgeschlagen ist?

Reply:
cakephp uses the function 
```php 
__('any text');
````
for localization.
I can generate a po file and translate it with poedit.
https://book.cakephp.org/3/en/core-libraries/internationalization-and-localization.html